### PR TITLE
Update README: Maintainers, Boards, Roadmap

### PR DESCRIPTION
**Pull request by junwoo091400**
### Solved Problem
Previously the Maintainer, Board compatibility & Roadmap were outdated

### Solution
1. Reorganize the maintainer list (need confirmation from individuals)
2. Update supported hardware list (mostly in sync with docs & checked link access)
3. Added note on outdated Roadmap

### Context
This is the first step after the community coordination call from January 30th: [Document Link](https://docs.google.com/document/d/1dnEmOeciJ4fdhNqPT9SZv9-5on6P-9ajKDf-wVV0B-A/edit?usp=sharing)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,21 +99,34 @@ This repository contains code supporting Pixhawk standard boards (best supported
...
+### Experimental
+
+These boards are nor maintained by PX4 team nor Manufacturer, and is not guaranteed to be compatible with up to date PX4 releases.
+
+  * [Raspberry PI with Navio 2](https://docs.px4.io/main/en/flight_controller/raspberry_pi_navio2.html)
+  * [Bitcraze Crazyflie 2.0](https://docs.px4.io/main/en/complete_vehicles/crazyflie2.html)
 
 ## Project Roadmap
 
+**Note: Outdated**
```
Comment (junwoo091400) [inline README.md:128]: TODO: Remove or update the roadmap to be updated (to be discussed)

**Comment by AlexKlimaj** [comment]
You can add the ARKV6X to the manufacturer supported board category. 
https://docs.px4.io/main/en/flight_controller/arkv6x.html

**Review by julianoes — COMMENTED** [review]
Nice!

**Comment by Jaeyoung-Lim** [comment]
@junwoo091400 Could you also add to the list for simulation?

File: README.md [hunk 2]
```diff
@@ -44,55 +44,50 @@ The PX4 Dev Team syncs up on a [weekly dev call](https://docs.px4.io/main/en/con
...
-    * [Markus Achtelik](https://github.com/markusachtelik)
-  * DDS/ROS2 Interface
-    * [Nuno Marques](https://github.com/TSC21)
+**Note: Being a Maintainer is an opt-in option, and must be coordinated with each person listed. This list that I (Junwoo) added doesn't reflect individual's consent at this stage**
+
+| Sector | Maintainer |
+|---|---|
+| Founder | [Lorenz Meier](https://github.com/LorenzMeier) |
+| Architecture | [Daniel Agar](https://github.com/dagar) / [Beat Küng](https://github.com/bkueng)|
+| State Estimation | [Mathieu Bresciani](https://github.com/bresch) / [Paul Riseborough](https://github.com/priseborough) |
```
Comment (junwoo091400) [inline README.md:53]: Approved by Mathieu

File: README.md [hunk 3]
```diff
@@ -44,55 +44,50 @@ The PX4 Dev Team syncs up on a [weekly dev call](https://docs.px4.io/main/en/con
...
+| Drivers | [Daniel Agar](https://github.com/dagar) |
+| QGroundControl | x |
+| Community QnA Call | [Ramon Roche](https://github.com/mrpollo) |
+| [Documentation](https://docs.px4.io/main/en/) | [Hamish Wilee](https://github.com/hamishwillee) |
+
+| Vehicle Type | Maintainer |
+|---|---|
+| Multirotor | [Matthias Grob](https://github.com/MaEtUgR) |
+| Fixed Wing | [Thomas Stastny](https://github.com/tstastny) |
+| VTOL | [Silvan Fuhrer](https://github.com/sfuhrer) |
```
Comment (junwoo091400) [inline README.md:64]: Approved by Silvan. Suggests splitting the group into 2: VTOL Control / VTOL Customization

Comment (sfuhrer) [reply README.md:64]: To be clear, I meant having VTOL and FW as separate categories, like proposed here. No need to further divide VTOL, but we should think about what context falls into "VTOL". And maybe start calling it "Hybrid VTOL" to separate it from all the multicopters and helicopters out there.

**Comment by swimmingseeds** [comment]
This pull request has been mentioned on **Discussion Forum, for PX4, Pixhawk, QGroundControl, MAVSDK, MAVLink**. There might be relevant details there:

https://discuss.px4.io/t/px4-community-q-a-february-15-2023/30744/2

File: README.md [hunk 4]
```diff
@@ -104,21 +99,34 @@ This repository contains code supporting Pixhawk standard boards (best supported
...
   * [Holybro Kakute F7](https://docs.px4.io/main/en/flight_controller/kakutef7.html)
-  * [Raspberry PI with Navio 2](https://docs.px4.io/main/en/flight_controller/raspberry_pi_navio2.html)
 
-Additional information about supported hardware can be found in [PX4 user Guide > Autopilot Hardware](https://docs.px4.io/main/en/flight_controller/).
+### Community supported
+
+These boards don't fully comply industry standards, and thus is solely maintained by the PX4 publc community members.
+
+  * [Hex Cube Orange](https://docs.px4.io/main/en/flight_controller/cubepilot_cube_orange.html)
+  * [Hex Cube Yellow](https://docs.px4.io/main/en/flight_controller/cubepilot_cube_yellow.html)
```
Comment (julianoes) [inline README.md:117]: I'm maintaining the Cubes, so these should be under "Manufacturer supported".

File: README.md [hunk 5]
```diff
@@ -44,55 +44,50 @@ The PX4 Dev Team syncs up on a [weekly dev call](https://docs.px4.io/main/en/con
...
+| Sector | Maintainer |
+|---|---|
+| Founder | [Lorenz Meier](https://github.com/LorenzMeier) |
+| Architecture | [Daniel Agar](https://github.com/dagar) / [Beat Küng](https://github.com/bkueng)|
+| State Estimation | [Mathieu Bresciani](https://github.com/bresch) / [Paul Riseborough](https://github.com/priseborough) |
+| OS/NuttX | [David Sidrane](https://github.com/davids5) |
+| Drivers | [Daniel Agar](https://github.com/dagar) |
+| QGroundControl | x |
+| Community QnA Call | [Ramon Roche](https://github.com/mrpollo) |
+| [Documentation](https://docs.px4.io/main/en/) | [Hamish Wilee](https://github.com/hamishwillee) |
```
Comment (julianoes) [inline README.md:58]:
```suggestion
| [Documentation](https://docs.px4.io/main/en/) | [Hamish Willee](https://github.com/hamishwillee) |
```

**Review by julianoes — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -104,21 +99,34 @@ This repository contains code supporting Pixhawk standard boards (best supported
     * [Pixhawk](https://docs.px4.io/main/en/flight_controller/pixhawk.html)
     * [Pixfalcon](https://docs.px4.io/main/en/flight_controller/pixfalcon.html)
 
-### Manufacturer and Community supported
+### Manufacturer supported
+
+These boards are maintained to be compatible with PX4-Autopilot by the Manufacturers.
+
```
Comment (AlexKlimaj) [inline README.md:105]: Add ARKV6X
* [ARK Electronics ARKV6X](https://docs.px4.io/main/en/flight_controller/arkv6x.html)

**Review by julianoes — APPROVED** [review]

**Review by MaEtUgR — DISMISSED** [review]
Thanks for addressing the readme. This pr makes it much better than before. We can iterate with additional changes 👍

**Comment by junwoo091400** [comment]
@mrpollo If you approve it, I will go ahead and merge :ok_hand:

**Review by sfuhrer — APPROVED** [review]

**Review by MaEtUgR — APPROVED** [review]

**Comment by junwoo091400** [comment]
Thank you everyone for the help making this happen!

**Pull request merged**